cpp/test/cpu_operations_test/Inverse_test.cc

```diff
@@ -62,7 +62,7 @@ TYPED_TEST(InverseTest, InverseFunctionality) {
   this->GetInverse();
   for (int i = 0; i < 3; i++) {
     for (int j = 0; j < 3; j++) {
-      EXPECT_EQ(this->output(i, j), this->correct(i, j));
+      ASSERT_NEAR(this->output(i, j), this->correct(i, j), 0.0001);
```

Comment:
Assert will cause the test exit for any assert failure, I suggest we use EXPECT_NEAR here.